backend/handlers/invite_collection_handler.py

```diff
@@ -34,36 +35,43 @@ def post(self, user):
         body = json.loads(self.request.body)
         data = body['data']
         host = self.request.host
-        type_of_invite = data.get('type_of_invite')
-
-        Utils._assert(type_of_invite == 'INSTITUTION',
-                      "invitation type not allowed", NotAuthorizedException)
-        
-        """TODO: Remove the assert bellow when the hierarchical invitations can be available
-        @author: Mayza Nunes 11/01/2018
-        """
-        Utils._assert(type_of_invite != 'USER',
-                      "Hierarchical invitations is not available in this version", NotAuthorizedException)
-
-        invite = InviteFactory.create(data, type_of_invite)
-
-        can_invite_inst = user.has_permission("send_link_inst_invite", invite.institution_key.urlsafe())
-        can_invite_members = user.has_permission("invite_members", invite.institution_key.urlsafe())
-    
-        if(can_invite_inst or can_invite_members):
-            institution = invite.institution_key.get()
+        invites_keys = []
+        invite = data['invite_body']
+        type_of_invite = invite.get('type_of_invite')
+        for email in data['emails']:
+            invite['invitee'] = email
+            Utils._assert(type_of_invite == 'INSTITUTION',
```

Comment:
Acho que todos os asserts poderiam estar fora do for, pois o tipo do convite sempre vai ser o mesmo para todos os convidados na lista de email, e o usuário que envia o convite também é o mesmo, para checar a permissão a instituição também é a mesma

Reply:
Bem lembrado